### PR TITLE
Update documentation font

### LINKED ISSUE #4848: Update web font
**Description of the desired feature**

The PyGMT website has a new font which is designed to maximize the difference between lowercase L and uppercase I, etc. Example here: https://www.pygmt.org/dev/api/generated/pygmt.Figure.grdimage.html#pygmt.Figure.grdimage

PR for PyGMT here: https://github.com/GenericMappingTools/pygmt/pull/938

It seems like GMT's site could benefit from the same update. Maybe also would be nice for a coherent "design language" for what it's worth.

Pinging @meghanrjones who was involved in the PyGMT font convo.

**Are you willing to help implement and maintain this feature?**

No

### PR DESCRIPTION
**Pull request by meghanrjones**
**Description of proposed changes**

This PR updates the font for the GMT documentation to have differences between lowercase L and uppercase I, etc (consistent with PyGMT). The Vercel bot (View deployment button) can be used to preview the changes in order to provide feedback about whether the font should be changed. The [PyGMT documentation](https://www.pygmt.org/dev/) also provides a reference point.

Fixes #4848 

**Reminders**

- [ ] Make sure that your code follows our style. Use the other functions/files as a basis.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Describe changes to function behavior and arguments in a comment below the function declaration.
- [ ] If adding new functionality, add a detailed description to the documentation and/or an example.

### CODE REVIEW

**Comment by meghanrjones** [comment]
Pinging @GenericMappingTools/gmt-contributors, @GenericMappingTools/core, and @GenericMappingTools/python for opinions on the GMT documentation font. You can click the "View Deployment" button to preview the documentation.

**Comment by meghanrjones** [comment]
My opinion is that the more distinct character styles in the font (in particular for I and l) improves the documentation.

**Comment by seisman** [comment]
Current documentation: https://docs.generic-mapping-tools.org/dev/

New documentation: https://gmt-n4txexgl8-gmt.vercel.app/

**Comment by liamtoney** [comment]
I'm getting different-looking headers...

Screen Shot 2021-03-02 at 12 53 27 PM

**Comment by liamtoney** [comment]
I feel this is a big boost in readability.

**Comment by weiji14** [comment]
| [Before (Lato)](https://docs.generic-mapping-tools.org/dev/std-opts.html) | [After (Atkinson Hyperlegible)](https://gmt-n4txexgl8-gmt.vercel.app/std-opts.html) |
|--|--|
| image | image|
| image | image |

File: doc/rst/_static/style.css [hunk 1]
```diff
@@ -8,6 +8,16 @@
     font-weight: normal;
 }
 
+body {
+	font-family: "Atkinson Hyperlegible", sans-serif;
+    font-size: 1.05em;
```
Comment (weiji14) [inline doc/rst/_static/style.css:13]: One small whitespace nitpick
```suggestion
    font-family: "Atkinson Hyperlegible", sans-serif;
    font-size: 1.05em;
```

**Comment by meghanrjones** [comment]
> I'm getting different-looking headers...
> 


Thanks, should be fixed now.

**Comment by KristofKoch** [comment]
I agree - readability is greatly improved with the new font.

File: doc/rst/_static/style.css [hunk 2]
```diff
@@ -1,38 +1,48 @@
...
+	font-weight: normal;
+}
+
+body {
+	font-family: "Atkinson Hyperlegible", sans-serif;
+	font-size: 1.05em;
+}
+
+h1, h2, h3, h4, h5, h6 {
+	font-family: "Atkinson Hyperlegible";
```
Comment (weiji14) [inline doc/rst/_static/style.css:17]: Shouldn't there be a fallback font here?

```suggestion
	font-family: "Atkinson Hyperlegible", sans-serif;
```

**Review by PaulWessel — APPROVED** [review]

**Pull request merged**